sdv/data_processing/data_processor.py

```diff
@@ -642,6 +649,9 @@ def reverse_transform(self, data, reset_keys=False):
         if not self.fitted:
             raise NotFittedError()
 
+        anonymized_data = pd.DataFrame()
```

Comment:
I think these are too far from where they are used. Initialize them in lines 680 and 687 instead.

Reply:
I forgot to remove them, nice catch